pdns/dnsdistdist/docs/reference/actions.rst

```diff
@@ -797,6 +797,9 @@ The following actions exist.
   Forge a response with the specified IPv4 (for an A query) or IPv6 (for an AAAA) addresses.
   If you specify multiple addresses, all that match the query type (A, AAAA or ANY) will get spoofed in.
 
+  Note that if you only specify addresses of one type (e.g. only IPv4 addresses), then queries for the other type (in this case AAAA queries), will **not** be spoofed.
+  This means that for most practical cases, you'd need to specify at least one IPv4 and one IPv6 address.
```

Comment:
I'm assuming this is correct, but... it isn't really helpful for at least the case I know of which is _wanting_ to not return an IPv6 address (or conversely not return an IPv4 address).

If `SpoofAction()` can't result in, e.g. `A` returning a certain set of answers and `AAAA` returning `NODATA`, then the help here should point to something that would be able to do that.

I know that dnsdist supports LUA: https://www.dnsdist.org/advanced/luaaction.html, and I ended up using LUA to achieve this (although I did it in pdns-auth).

I'm not sure if there's something else that would work, but ideally the help would point to the best practice for that effect.

Reply:
What you can do is use a selector to only call `SpoofAction` for `A` and instead call `SetNegativeAndSOAAction`[1] for `AAAA`

[1]: https://www.dnsdist.org/reference/actions.html#SetNegativeAndSOAAction

Reply:
> What you can do is use a selector to only call `SpoofAction` for `A` and instead call `SetNegativeAndSOAAction`[1](https://www.dnsdist.org/reference/actions.html#SetNegativeAndSOAAction) for `AAAA` and other types.

Reply:
Thanks. I'll update the PR to offer a suggestion to use SetNegateiveAndSOAAction for AAAA.